src/lib/addons/feature-event-formatter-md.ts

```diff
@@ -140,6 +142,18 @@ const EVENT_MAP: Record<string, IEventData> = {
         action: '*{{user}}* sent to review change request {{changeRequest}}',
         path: '/projects/{{event.project}}/change-requests/{{event.data.changeRequestId}}',
     },
+    [CHANGE_REQUEST_SCHEDULED]: {
+        action: '*{{user}}* scheduled change request {{changeRequest}} in project *{{event.project}}*',
+        path: '/projects/{{event.project}}/change-requests/{{event.data.changeRequestId}}',
+    },
+    [CHANGE_REQUEST_SCHEDULED_APPLICATION_SUCCESS]: {
+        action: '*{{user}}* successfully applied scheduled change request {{changeRequest}} in project *{{event.project}}*',
+        path: '/projects/{{event.project}}/change-requests/{{event.data.changeRequestId}}',
+    },
+    [CHANGE_REQUEST_SENT_TO_REVIEW]: {
+        action: '*{{user}}* failed to apply scheduled change request {{changeRequest}} in project *{{event.project}}*',
```

Comment:
Because this is an urgent event, maybe we should put the`failed` bit first? Not very important, just a suggestion.


```suggestion
        action: '*Failed* to apply the scheduled change request {{changeRequest}} by *{{user}}* in project *{{project}}*.',
```